src/templates/script/ScriptBundleTemplateProvider.ts

```diff
@@ -35,17 +36,12 @@ export class ScriptBundleTemplateProvider extends ScriptTemplateProvider {
         const bundleMetadata: IBundleMetadata | undefined = await this.getBundleInfo();
         const release: bundleFeedUtils.ITemplatesRelease = await bundleFeedUtils.getRelease(bundleMetadata, latestTemplateVersion);
 
-        const bindingsRequest: requestUtils.Request = await requestUtils.getDefaultRequestWithTimeout(release.bindings);
-
         const language: string = this.getResourcesLanguage();
         const resourcesUrl: string = release.resources.replace('{locale}', language);
-        const resourcesRequest: requestUtils.Request = await requestUtils.getDefaultRequestWithTimeout(resourcesUrl);
-
-        const templatesRequest: requestUtils.Request = await requestUtils.getDefaultRequestWithTimeout(release.functions);
 
         [this._rawBindings, this._rawResources, this._rawTemplates] = <[object, object, object[]]>(
-            await Promise.all([bindingsRequest, resourcesRequest, templatesRequest].map(requestUtils.sendRequest))
-        ).map(parseJson);
+            await Promise.all([release.bindings, resourcesUrl, release.functions].map(url => requestUtils.sendRequestWithTimeout({ method: 'GET', url })))
```

Comment:
I don't know if it's worth condensing all of this just to look cleaner.  For instance, I don't think I'd realize `release.functions` is the url to retrieve templates.

Reply:
Didn't do it to look cleaner. Just did it because the new `requestUtils` only has one method (`sendRequestWithTimeout`) whereas the old `requestUtils` had two (`getDefaultRequestWithTimeout` and `sendRequest`)